package.json

```diff
@@ -17,6 +17,7 @@
     "vue-router": "^4.1.6"
   },
   "devDependencies": {
+    "@gradin/tailwindcss-scrollbar": "^3.0.1",
```

Comment:
what's that?

Reply:
https://www.npmjs.com/package/@gradin/tailwindcss-scrollbar

This is some plugin for Tailwind that allows you to style your scrollbar so it doesn't look like this default one.